src/core/xref.js

```diff
@@ -72,6 +87,50 @@ export async function run(conf, elems) {
   addDataCiteToTerms(results, xrefMap, conf);
 }
 
+/**
+ * converts conf.xref to object with url and spec properties
+ */
+function normalizeConfig(conf) {
+  const xrefDefaults = {
+    url: API_URL,
+    profile: "",
+    specs: null,
+  };
+
+  const normalizedConfig = {};
+  const type = Array.isArray(conf.xref) ? "array" : typeof conf.xref;
+  switch (type) {
+    case "boolean":
+      // ... true
+      Object.assign(normalizedConfig, xrefDefaults);
+      break;
+    case "string":
+      if (profiles[conf.xref]) {
+        // We have the profile
+        Object.assign(normalizedConfig, xrefDefaults, {
+          specs: profiles[conf.xref],
+        });
+      } else {
+        pub("error", "Invalid cross-references profile");
+        Object.assign(normalizedConfig, xrefDefaults);
+      }
+      break;
+    case "array":
+      Object.assign(normalizedConfig, xrefDefaults, {
+        specs: [].concat(conf.xref),
+      });
+      break;
+    case "object":
+      Object.assign(normalizedConfig, xrefDefaults, conf.xref);
+      break;
+    default:
+      // show user error - bad configuration
+      Object.assign(normalizedConfig, xrefDefaults);
+      pub("error", "Invalid cross-references configuration");
+  }
+  conf.xref = normalizedConfig;
```

Comment:
Don't reassign it, as it's now an internal representation. 

```suggestion
  return normalizedConfig;
```

Then use normalizedConfig in `run()`